### ISSUE TITLE
<model-viewer> doesn't recognize our WebXR Hit Test API

### ISSUE THREAD

**Issue by takahirox** [MozillaReality/webxr-ios-js#83]
Google's [&lt;model-viewer&gt; AR example](https://modelviewer.dev/examples/augmented-reality.html) doesn't recognize ~our API~ our WebXR Hit test API even with #82.

They seems to check Hit Test API availability with `window.XRSession.prototype.requestHitTestSource !== undefined` before calling `navigator.xr.isSessionSupported()`.

But our shim only defines the classes and doesn't define their methods. So `.requestHitTestSource` will be undefined then they determines that our browser doesn't support AR hit test API.

Maybe related #77

**Comment by blairmacintyre** [comment]
that sounds like a bug in their code;  perhaps submit an issue there?  They are clearly checking for a non-standard API since Google is the only one shipping an unreleased API.

**Comment by takahirox** [comment]
I think I wrote poorly. I updated the issue comment and title. 

They check 1 WebXR API availability and 2. WebXR Test Hit API availability.

https://github.com/google/model-viewer/blob/master/packages/model-viewer/src/constants.ts#L19-L23

```
export const HAS_WEBXR_DEVICE_API = navigator.xr != null &&
    self.XRSession != null && navigator.xr.isSessionSupported != null;

export const HAS_WEBXR_HIT_TEST_API =
    HAS_WEBXR_DEVICE_API && self.XRSession!.prototype.requestHitTestSource;
```

The shim defines the classes but doesn't define methods except for `navigator.xr.isSessionSupported/requestSession` so even with #82 `HAS_WEBXR_HIT_TEST_API` will be false because `.XRSession.prototype.requestHitTestSource` is undefined before loading the entire polyfill.
